.github/CODEOWNERS

```diff
@@ -4,12 +4,12 @@
 *                                   @DataDog/apm-core-python
 
 # Framework Integrations
-ddtrace/ext/                        @DataDog/apm-core-python @DataDog/apm-framework-integrations
-ddtrace/contrib/                    @DataDog/apm-core-python @DataDog/apm-framework-integrations
-ddtrace/internal/schema/            @DataDog/apm-core-python @DataDog/apm-framework-integrations
-tests/contrib/                      @DataDog/apm-core-python @DataDog/apm-framework-integrations
-tests/internal/peer_service         @DataDog/apm-core-python @DataDog/apm-framework-integrations
-tests/internal/service_name         @DataDog/apm-core-python @DataDog/apm-framework-integrations
+ddtrace/ext/                        @DataDog/python-guild @DataDog/apm-framework-integrations
```

Comment:
I can't tell if this should be integrations + sdk? or just integrations?

I do see some CI Viz stuff in there, but I think `ddtrace.ext` is meant to be public facing (ish) for tracing constants... sort of...

Reply:
probably ADM + IDM for now